autoscaler/controllers/datacollection/configmap.go

```diff
@@ -184,8 +231,26 @@ func calculateConfigMapData(apps *odigosv1.InstrumentedApplicationList, dests *o
 			},
 		},
 		Service: config.Service{
-			Pipelines:  map[string]config.Pipeline{},
+			Pipelines:  map[string]config.Pipeline{
+				"metrics/otelcol": {
+					Receivers: []string{"prometheus"},
+					Processors: []string{"resource/pod-name"},
+					Exporters: []string{"otlp/ui"},
```

Comment:
As the traffic data is only relevant in realtime, maybe we should configure the otlp exporter to drop data if UI is not responding? Consider configuring `retry_on_failure`,`sending_queue`,`timeout`:
https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md

Reply:
disabled retry